src/keria/app/agenting.py

```diff
@@ -289,6 +290,8 @@ def __init__(self, hby, rgy, agentHab, agency, caid, **opts):
         self.agentHab = agentHab
         self.agency = agency
         self.caid = caid
+        self.cfd = MappingProxyType(dict(self.hby.cf.get()) if self.hby.cf is not None else dict())
```

Comment:
Why do we need MappingProxyType here instead of accessing the dict directly?

Reply:
I've used it to make the `dict` immutable, so it cannot be changed unintentionally.